### ISSUE TITLE
Different username than 'pi' breaks Klipper

### ISSUE THREAD

**Issue by vordenken** [guysoft/OctoKlipperPi#11]
```
Apr 16 18:25:52 octopi systemd[1]: Starting LSB: Klipper daemon...
Apr 16 18:25:53 octopi klipper[375]: Starting klipper: klipper
Apr 16 18:25:53 octopi klipper[479]: start-stop-daemon: unable to stat /home/pi/klippy-env/bin/python (No such file or directory)
Apr 16 18:25:53 octopi klipper[480]:  failed!
Apr 16 18:25:53 octopi systemd[1]: Started LSB: Klipper daemon.
```

**Comment by guysoft** [comment]
Yep thats a bug.
its becaue this code needs to be changed from ``pi`` to ``${BASE_USER}`` and ill have to look on to it
https://github.com/guysoft/OctoKlipperPi/blob/main/src/scripts/03-install-klipper#L30

**Comment by jakebullet70** [comment]
If this has not been addressed, I am getting ready to convert a Rock 64 PI over to Octoklipper. I run the DietPi OS and will install Octoprint from their menu system. So if someone wants some testing... :)

**Comment by guysoft** [comment]
You are welcome to try. I am putting my effort ATM to add support on CustomPiOS for features that should make extra hardware support much much easier (meta or super modules, and remote modules). So if you write something that adds support to Rock 64 Pi it should make it easier to add support later on.

It should make these easier to use:
https://github.com/mainsail-crew/MainsailOS/tree/feat/lepotato/config

Back to topic on this issue, I just realized it might be easier than previously thought to fix it. I just pushed a fix, if anyone could test the output from the github action that would be helpful!
